SWLOR.Game.Server/Feature/PlayerInitialization.cs

```diff
@@ -272,6 +272,9 @@ private static void InitializeLanguages(uint player, Player dbPlayer)
                 case RacialType.Nautolan:
                     languages.Add(SkillType.Nautila);
                     break;
+                case RacialType.Ewok:
+                 languages.Add(SkillType.Ewokese);
```

Comment:
There are some formatting issues here which reduce readability. Make sure these lines (276 and 277) are formatted with tabs like the previous entries are set up.